lxd/api.go

```diff
@@ -184,6 +184,11 @@ func restServer(d *Daemon) *http.Server {
 	}
 
 	for _, c := range api10 {
+		// Every 1.0 endpoint should have a type for the API metrics.
+		if !shared.ValueInSlice(c.MetricsType, entity.APIMetricsEntityTypes()) {
```

Comment:
why call entity.APIMetricsEntityTypes() for every API endpoint in the loop? Better to store in temp var right?